ui/components-react/src/test/common/DelayedSpinner.test.tsx

```diff
@@ -17,17 +16,13 @@ describe("<DelayedSpinner />", () => {
   });
 
   it("renders spinner with delay", async () => {
-    const clock = sinon.useFakeTimers({ now: Date.now() });
-    const delay = 100;
-    const { container } = render(<DelayedSpinner delay={delay} />);
+    const { container } = render(<DelayedSpinner delay={100} />);
     expect(container.children.length).to.be.eq(0);
```

Comment:
Using sinon.useFakeTimers changes behavior of other tests unpredictably and I couldn't find a way to properly dispose of it in a way that it wouldn't impact other tests. Therefore, I decided to get rid of it altogether. 

That being said, removing it in this test might make it flaky, so I am open to suggestions on how to remedy this

Reply:
I assume you tried this example: https://github.com/sinonjs/fake-timers?tab=readme-ov-file#faking-the-native-timers? Seems unlikely it's a problem with the library, based on the fact that it has 9M weekly downloads and no related issue in its repo.

Reply:
Changed the tests that use `sinon.useFakeTimers` to only initialize once per test suite and it seems like it fixed the issue.

Reply:
Should probably change it for other tests also, so that it wouldn't occur again...